core/net/src/hybrid/rest_api.rs

```diff
@@ -0,0 +1,28 @@
+use actix_web::{web::Data, HttpResponse, Responder, Scope};
+use ya_client_model::net::{Status, NET_API_V2_NET_PATH};
+
+use crate::error::Result;
+
+use super::client::ClientProxy;
+
+pub fn web_scope() -> Scope {
+    let scope = actix_web::web::scope(NET_API_V2_NET_PATH);
+    match ClientProxy::new() {
+        Ok(client) => scope.app_data(Data::new(client)).service(get_info),
```

Comment:
You can also create instances of `ClientProxy` in handlers (it's not expensive)

Reply:
In REST endpoint handlers? It was the initial idea. Instead task evolved into initializing ClientProxy in Services::gsb and doing some checks to call it before Services::rest

Reply:
Moved back to creating `ClientProxy` in REST endpoint handlers, because the race condition seems to be caused by thread spawned in [Net::gsb](https://github.com/golemfactory/yagna/blob/master/core/net/src/hybrid/service.rs#L80). I am waiting for test results - this time should finally pass on GitHub too.